Realm/RLMSchema.mm

```diff
@@ -39,10 +40,29 @@ @interface RLMSchema ()
 @end
 
 static RLMSchema *s_sharedSchema;
+static RLMSchema *s_partialSharedSchema;
 static NSMutableDictionary *s_localNameToClass;
 
 @implementation RLMSchema
 
++ (instancetype)schemaWithObjectClasses:(NSArray *)classes {
+    NSUInteger count = classes.count;
+    Class *classArray = static_cast<Class *>(malloc(count * sizeof(Class)));
```

Comment:
This is never freed. Can just do `auto classArray = std::make_unique<Class[]>(count);`.